### DISCUSSION TITLE
My service says 0.2.1 - in HACS it's 0.5.2. Do I have the most recent version?

### DISCUSSION THREAD

**Discussion by scooterama** [kvj/hass_nuki_ng#192] [Q&A]
My service says version 0.2.1 - in HACS it's 0.5.2. Do I have the most recent version?

I manually installed the first version - now I uninstalled, rebooted, installed via HACS, rebooted, installed the integration, rebooted and still 0.2.1

**Comment by gnoby** [comment]
I have the same problem. Did you find out if you have the right version?
